services/web-app/components/design-kit-catalog-item/design-kit-catalog-item-meta/design-kit-catalog-item-meta.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Scales } from '@carbon/react/icons'
```

Comment:
this seems very similar to `AssetCatalogItemMeta`, is there an opportunity to convert that to `CatalogItemMeta` and reuse it?

Reply:
done!